src/frame/index.ts

```diff
@@ -71,7 +71,7 @@ function onParentPortMessage(data: RequestMessage, callback: (message: ResponseM
                 data.message.documentName,
                 data.message.userGrants,
                 data.message.groupGrants,
-                data.message.grantToAuthor
+                data.message.grantToAuthor !== false
```

Comment:
We should put comments on these things. I was thinking:
```
Here we check to see if this is not false because versions < 3.1 send nothing in this spot so we have to default 
the value to ensure the types are actually correct. Note that the types >= 3.1 are actually still consistent as well, 
we just resolve the optionality in the shim.
```

Reply:
Done